x-pack/agent/docs/agent_configuration_example.yml

```diff
@@ -0,0 +1,209 @@
+outputs:
+  default:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
```

Comment:
Defining outputs separately means that users can't specify that datasources send to separate outputs. What was the previous thought on including an output per datasource?

Reply:
They can still decide where a _datasource_ is sending the data, by using the `use_output: long_term_storage`, we wanted to simplify it by defining a single way to create outputs but allow people to reuse them with their defined names.

Reply:
@mostlyjason https://github.com/elastic/beats/pull/15940/files#diff-043e80bbcc20fded11325ef31433396bR60 in action